### DISCUSSION TITLE
Items with light components add the 'light' tag to normal instead of predefined tags

### DISCUSSION THREAD

**Discussion by Jade-Harleyy** [FakeFishGames/Barotrauma#13968] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

When placing an item that has a LightComponent into the sub editor, it automatically adds the `light` tag to the item's _normal_ tags instead of its predefined tags, allowing it to be removed.

### Reproduction steps

1. Open the submarine editor.
2. Place a "Light Component" item.
3. Observe the item's tags.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Other

### -

Submarine Editor

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/pull/5345/commits/53088bead6f53f414bd4b3685957f9dcfde5144f

**Comment by Rokvach** [comment]
Tested, working correctly. Closing.
